src/Desktop/Types/TextAttributeType.cs

```diff
@@ -17,6 +18,7 @@ namespace Axe.Windows.Desktop.Types
     public class TextAttributeType : TypeBase
     {
 #pragma warning disable CA1707 // Identifiers should not contain underscores
+        [NotType]
```

Comment:
This is awkward--we're declaring a list of types but saying that this isn't a type. It would probably help if the attribute had a more specific name--name `NotChangeEventType` (or something similar) instead of `NotType`